WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -331,24 +328,27 @@ private extension ShippingLabelFormViewController {
         } onButtonTouchUp: { [weak self] in
             guard let self = self else { return }
             ServiceLocator.analytics.track(.shippingLabelPurchaseFlow, withProperties: ["state": "purchase_initiated",
-                                                                                        "amount": self.viewModel.selectedRate?.rate ?? 0,
-                                                                                        "fulfill_order": self.shouldMarkOrderComplete])
+                 "amount": self.viewModel.selectedRates.map { $0.totalRate }
+                                                       .reduce(0, +),
```

Comment:
Nit: I think it'd be better if we extract this into a variable so that we don't have to repeat the calculation in several places?